Standartise evaluator pages (#1943)
spryker/spryker-docs: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/multidimensional-array.md

If a plugins stack is used on the project level, not all structures are necessarily required. Deeply nested multidimensional arrays make configuration hard to upgrade.
This check verifies that multidimensional arrays have a maximum of two levels of nesting inside.

## Example of an evaluator error message

```bash
======================
MULTIDIMENSIONAL ARRAY
======================

Message: Reached max level of nesting for the plugin registration in the {FormDependencyProvider::getPlugins()}.

>          The maximum allowed nesting level is 2. Please, refactor code, otherwise it will cause upgradability issues in the future.

Reviewer: Avoid predicting the future.
```suggestion
         The maximum allowed nesting level is 2. Refactor the code; otherwise, it can cause upgradability issues in the future.
```
